components/FaceScanner.tsx

```diff
@@ -81,14 +82,16 @@ export const FaceScanner: React.FC<FaceScannerProps> = props => {
 
   return (
     <Column fill align="space-between">
-      <View style={Theme.Styles.scannerContainer}>
+      <View style={Theme.CameraEnabledStyles.scannerContainer}>
         <Camera
-          style={Theme.Styles.scanner}
+          {...testIDProps('camera')}
+          style={Theme.CameraEnabledStyles.scanner}
           type={whichCamera}
           ref={setCameraRef}
         />
       </View>
       <Text
+        testID="imageCaptureGuide"
```

Comment:
Why are we not using testIDProps?

Reply:
@sree96 We are using Custom Text component and inside that we are using testIDProps **{...testIDProps(props.testID)}**